src/erc721/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Nft } from '../../generated/schema'
+import { Transfer } from '../../generated/ERC721-Chronos-Traveler/ERC721'
+import { generateNftId, createNft, findOrCreateAccount } from '../utils/helper'
+
+export function handleErc721Transfer(event: Transfer): void {
```

Comment:
We were following a pattern: `event-name-in-pascal-case-handler` for the file and `handleEventName` for the handler. Exporting everything in a `index.ts`

```suggestion
export function handleTransfer(event: Transfer): void {
```